### PR TITLE
Align docs colours

### LINKED ISSUE #338: Dedent codeblocks in the docs
Some example code blocks contain unnecessary indentation. [Example](https://github.com/bakdata/kpops/blob/main/docs/docs/user/references/editor-integration.md?plain=1#L18-L22)

Note that this also happens with snippets:

```
    ```json
      --8<--
      ./docs/resources/editor_integration/settings.json
      --8<--
    ```
```
leads to
![image](https://github.com/bakdata/kpops/assets/44573859/e2656092-4836-4284-9cb6-ca3e0bf2d7a4)

Instead we want the following:

```
    ```json
    --8<--
    ./docs/resources/editor_integration/settings.json
    --8<--
    ```
```
![image](https://github.com/bakdata/kpops/assets/44573859/a5cc7a9b-998f-47f0-825c-1adce08b0e0d)

### PR DESCRIPTION
**Pull request by raminqaf**
Fixes #338

### CODE REVIEW

File: docs/docs/user/references/editor-integration.md [hunk 1]
```diff
@@ -17,9 +17,9 @@ KPOps provides JSON schemas that enable autocompletion and validation for some o
 ??? note "`settings.json`"
 
     ```json
-      --8<--
-      ./docs/resources/editor_integration/settings.json
-      --8<--
+    --8<--
+    ./docs/resources/editor_integration/settings.json
+    --8<--
```
Comment (disrupted) [inline docs/docs/user/references/editor-integration.md:22]: is this the correct indentation now?

File: docs/docs/stylesheets/extra.css [hunk 2]
```diff
@@ -0,0 +1,14 @@
+:root > * {
+    --md-primary-fg-color: #a599ff;
+    --md-accent-fg-color: #B0A5FF;
```
Comment (disrupted) [inline docs/docs/stylesheets/extra.css:3]: the first one is the "main KPOps color", right? What's the second one? Is it a slightly darker shade? consider adding a comment here for explanation

**Review by irux — APPROVED** [review]
LGTM

File: docs/docs/overrides/home.html [hunk 3]
```diff
@@ -13,25 +13,16 @@
...
-    }
-
-    /* Remove spacing, as we cannot hide it completely */
+    /*!* Remove spacing, as we cannot hide it completely *!*/
     .md-main__inner {
         margin: 0;
     }
 
-    /* Hide main content for now */
+    /*!* Hide main content for now *!*/
```
Comment (disrupted) [inline docs/docs/overrides/home.html:25]: what's the meaning of this `*!*`?

File: docs/docs/user/references/editor-integration.md (hunk 1 above)
Comment (disrupted) [reply docs/docs/user/references/editor-integration.md:22]: do you want to reference the issue in this PR or is there further work required to close it?

**Pull request merged**